homeassistant/components/zwave_js/sensor.py

```diff
@@ -121,3 +115,13 @@ def unit_of_measurement(self) -> str:
             return TEMP_FAHRENHEIT
 
         return self.info.primary_value.metadata.unit
+
+    @property
+    def device_state_attributes(self):
+        """Return the device specific state attributes."""
+        if self.info.primary_value.metadata.states and self.info.primary_value.value is not None:
+            # add the value's label as property for multi-value (list) items
+            label = self.info.primary_value.metadata.states.get(
+                self.info.primary_value.value
+            ) or self.info.primary_value.metadata.states.get(str(self.info.primary_value.value))
```

Comment:
Why do we need to try again with a string lookup?

Reply:
because I've seen examples where the key was a string

Reply:
guess that's what's happend with user defined config files

Reply:
Ok. Longterm, maybe we need to add validation to the Python library? It's bad if we can't rely on the specs in the docs.

Reply:
these values come from the user configuration files so there's a small chance of these inconsistencies.
probably we should convert the key to int when we receive it in the lib ?

Reply:
Yeah, the more consistent the library output is, the better.

Reply:
do you want me to address that now in the lib or later ?

Reply:
Later. :+1: